src/patternfly/components/LogViewer/log-viewer.scss

```diff
@@ -21,11 +24,14 @@
   --pf-c-log-viewer--m-line-numbers__main--before--BackgroundColor: var(--pf-global--BorderColor--100);
 
   // List
-  // --pf-c-log-viewer__list--PaddingTop: var(--pf-global--spacer--sm);
-  // --pf-c-log-viewer__list--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-log-viewer__list--Width: 100%;
```

Comment:
Is it better to set the width to `unset` or something similar here? If I set to 100%, there are still some problems when scrolling to the right.

<img width="828" alt="Screen Shot 2021-11-08 at 1 10 27 PM" src="https://user-images.githubusercontent.com/37624318/140795272-c610c09c-79f6-49ed-b268-4eb8b45bb943.png">